include/espeak.md

```diff
@@ -31,7 +31,8 @@ Modifications will need to be made in `<nvda repo root>/nvdaHelper/espeak`
    1. Modify the `<nvda repo root>/nvdaHelper/espeak/config.h` file as required.
 1. Update our record of the version number and build.
    1. Change back to the NVDA repo root
-   1. Update the package version in `<nvda repo root>/nvdaHelper/espeak/config.h`
+   1. Update the `/DPACKAGE_VERSION` in `espeak/sconscript`
+      - The preprocessor definition is used to create `<nvda repo root>/nvdaHelper/espeak/config.h`

```

Comment:
I don't think this statement is quite correct. The preprocessor definitions are specified on the commandline  **instead of** in config.h. config.h is empty and stays empty.

Reply:
Yes, "instead of" would be more clear. I guess config.h still has to exist, even if empty.